### PR TITLE
Implement feed

### PR DESCRIPTION
**Pull request by balderfer**
You can now create new posts and load more as you scroll down

### CODE REVIEW

File: public/javascripts/src/actions/FeedActionCreators.jsx [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+var ActionTypes = LoungeConstants.ActionTypes;
+
+module.exports = {
+
+  loadPosts: function() {
+    AppDispatcher.dispatch({
+      type: ActionTypes.LOAD_POSTS
+    })
+  }
+}
```
Comment (Roystbeef) [inline public/javascripts/src/actions/FeedActionCreators.jsx]: add newline pls

Comment (Roystbeef) [inline public/javascripts/src/actions/FeedActionCreators.jsx]: also, why don't we set module.exports equal to the function over an object with the loadPosts function?

File: public/javascripts/src/actions/ServerActionCreators.jsx [hunk 2]
```diff
@@ -11,5 +11,14 @@ module.exports = {
...
+
+  receivePosts: function(json, errors) {
+    AppDispatcher.dispatch({
+      type: ActionTypes.RECEIVE_POSTS,
+      json: json,
+      errors: errors
+    })
   }
+
 }
```
Comment (Roystbeef) [inline public/javascripts/src/actions/ServerActionCreators.jsx]: new line pls

File: public/javascripts/src/actions/ServerActionCreators.jsx [hunk 3]
```diff
@@ -11,5 +11,14 @@ module.exports = {
...
       errors: errors
     })
+  },
+
+  receivePosts: function(json, errors) {
+    AppDispatcher.dispatch({
+      type: ActionTypes.RECEIVE_POSTS,
+      json: json,
+      errors: errors
+    })
```
Comment (Roystbeef) [inline public/javascripts/src/actions/ServerActionCreators.jsx]: semicolon

File: public/javascripts/src/components/feed/CreatePost.jsx [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+              <textarea type="text" className="form-control" id="createNewPostContent" ref="content" placeholder="What's on your mind?" />
+            </div>
+            <button type="submit" className="btn btn-primary">Submit</button>
+          </form>
+        </div>
+      </div>
+    );
+  }
+
+});
```
Comment (Roystbeef) [inline public/javascripts/src/components/feed/CreatePost.jsx]: newline

**Comment by Roystbeef** [comment]
we should probably be consistent with our filenames. I know at picnichealth, we had all of our files in snake_case. Also similarly, we should be consistent `module.exports = React.createClass` vs `var Feed = React.createClass ... module.exports = Feed;`.

File: public/javascripts/src/utils/WebAPIUtils.js [hunk 5]
```diff
@@ -39,7 +42,9 @@ module.exports = {
       })
       .set('Accept', 'application/json')
       .end(function(error, res) {
-        if (res) {
+        if (error) {
+          console.log(error);
```
Comment (Roystbeef) [inline public/javascripts/src/utils/WebAPIUtils.js]: These should probably be `console.error(error)`

**Comment by Roystbeef** [comment]
Uhh, yeah so I just pointed out some syntax stuff. Function wise, You're gonna have to explain to me what this code does in person probably, since a lot of it is going over my head rn.

File: public/javascripts/src/actions/FeedActionCreators.jsx (hunk 1 above)
Comment (balderfer) [inline public/javascripts/src/actions/FeedActionCreators.jsx]: Because as the feed view becomes more complex we will have more functions here

**Pull request merged**